DOCS-7165 made edits (#278)
mulesoft/docs-api-manager: modules/ROOT/pages/environment-permission-task.adoc

include::_attributes.adoc[]
endif::[]

> To be able to fully access API Manager operations within an environment, users must have the API Manager Environment Administrator permission. An API Manager environment administrator role is associated to tasks performed only in API Manager. You can change permissions for API Manager-related operations from Anypoint Access Management.

Reviewer: ```suggestion
To be able to fully access API Manager operations within an environment, users must have the API Manager Environment Administrator permission. An API Manager Environment Administrator role is associated with tasks performed only in API Manager. You can change permissions for API Manager-related operations from Anypoint Access Management.
```

Author: Done! Thanks!

Regarding roles, note that:

* Roles and permissions assigned to a user in one particular business group are independent of the roles and permissions assigned to that same user in other business groups. 

> * Roles are business group specific, so ensure that you are in the correct business group for which to manage roles by clicking the menu next to your username on the top-right of the screen.

Reviewer: ```suggestion
* Roles are business group-specific, so ensure that you are in the correct business group for which to manage roles by clicking the menu next to your username on the top-right of the screen.
```

Author: Done! Thanks!

> The following diagram explains the tasks covered by each permission:

Reviewer: ```suggestion
The following diagram illustrates the tasks enabled by having each permission:
```

Author: Done! Thanks!

image::apim-permissions-relationships.png[permission relationships]

> In the above diagram, an Environment Administrator has access to all the functionalities and permissions.

Reviewer: ```suggestion
In this diagram, note that an Environment Administrator has access to all the functionalities and permissions.
```

Author: Done! Thanks!

Outer permissions provide permission to the inner permissions. For example, Manage Alerts gives permission to View Alerts and to View API Configuration.

API Manager supports the following additional environment-level permissions:

[%header,cols="40a,15a,60a"]
|===
| Permission | Description | Notes
| Deploy API Proxies | Deploy or redeploy an API proxy | Requires Runtime Manager and

> Hybrid access. This role enables deployment access from the API Manager, but cannot

Reviewer: ```suggestion
Hybrid access. This role enables deployment access from API Manager, but cannot
```

Author: Done! Thanks!

> guarantee that deployment is successful due to its dependency on Runtime Manager or Hybrid.

Reviewer: "Hybrid"? What is the full name?

Author: Done! Thanks!

| Manage Alerts | Create, edit, and delete alerts. 
| Manage APIs Configuration | Import APIs from Exchange | This permission also
enables a user to promote an API to another environment, edit the configuration of an API, and delete
APIs.
+
[NOTE]
--

> This permission is per environment. API visibility can’t be restricted to specific users inside your Organization

Reviewer: ```suggestion
This permission is per environment. API visibility can’t be restricted to specific users inside your organization.
```

Author: Done! Thanks!

--
| Manage Contracts | Deploy, reject, and revoke contracts | This permission also gives the user permission over tiers.
| Manage Policies | Create, edit, delete, enable, disable, and reorder a policy 
| View Alerts | List alerts and view configuration summary of an alert 

> | View APIs Configuration | List APIs, view configuration summary of an API, and download proxies.

Reviewer: These three rows have only two columns?

Author: Done! Thanks!

+
[NOTE]
--
This permission is per environment. API visibility can’t be restricted to specific users inside your Organization
--
| View Contracts | List contracts and their configuration | Can also list SLA tiers and their configuration.

> | View Policies | View a listing of policies and view the configuration of a policy | 

Reviewer: ditto?

Author: Done! Thanks!

> == Granting Permissions

Reviewer: ```suggestion
== Grant Permissions
```

Reviewer: Where is the intro to the steps?

Author: Done! Thanks!

. *Click Access Management* > *Users* > *_a user to give permissions_* > *API Manager* > Permissions.

> . Select the permissions for the user and click the blue plus icon image:blue-plus.png[blue plus icon] to save the permissions:

Reviewer: ```suggestion
. Select the permissions for the user and click image:blue-plus.png[blue plus icon] to save the permissions:
```

Author: Done! Thanks!
